cypress/utils/utils.ts

```diff
@@ -826,3 +827,37 @@ export function selectWithinModal(selector: string): void {
         click(selector);
     });
 }
+
+export function selectWithin(parent, selector: string): void {
+    cy.get(parent).within(() => {
+        click(selector);
+    });
+}
+
+//function to select checkboxes
+export function selectCheckBox(selector: string): void {
+    cy.get(selector, { timeout: 120 * SEC }).then(($checkbox) => {
+        if (!$checkbox.prop("checked")) {
+            click(selector);
+        }
+    });
+}
+
+//function to unselect checkboxes
+export function unSelectCheckBox(selector: string): void {
+    cy.get(selector, { timeout: 120 * SEC }).then(($checkbox) => {
+        if ($checkbox.prop("checked")) {
+            click(selector);
+        }
+    });
+}
+
+export function applyAction(itemName: string, action: string): void {
```

Comment:
There is already performRowAction for this purpose in utils .

Reply:
Found it, I prefer my variant, less parameters, no conditions inside...

Reply:
You can make the same method simpler. We cannot keep two methods for same thing . That will be a lot of unnecessary code . Whole code needs to use one method for common things.

Reply:
My `applyAction` is removed, its content moved to your `performRowAction`. Validated that actions in `stakeholder` continue to function properly.